assets/js/app.js

```diff
@@ -555,7 +596,7 @@ function setupPlayground() {
         errorMessage = `An error occurred: ${error.message}`;
       }
 
-      $('#inference_response_raw').text(errorMessage);
+      alert(errorMessage);
```

Comment:
I think it may be better to show errors in the UI instead of having an `alert` popup.